tools/pipelines/templates/build-npm-package.yml

```diff
@@ -1,598 +1,540 @@
+                        exit -1;
+                      fi
+                    fi
+              - template: /tools/pipelines/templates/include-use-node-version.yml@self
+              - template: /tools/pipelines/templates/include-install.yml@self
+                parameters:
+                  packageManager: '${{ parameters.packageManager }}'
+                  buildDirectory: '${{ parameters.buildDirectory }}'
+                  packageManagerInstallCommand: '${{ parameters.packageManagerInstallCommand }}'
+              - '${{ if eq(parameters.taskPublishBundleSizeArtifacts, false) }}':
+                  - template: >-
+                      /tools/pipelines/templates/include-set-package-version.yml@self
+                    parameters:
+                      buildDirectory: '${{ parameters.buildDirectory }}'
+                      buildNumberInPatch: '${{ parameters.buildNumberInPatch }}'
+                      buildToolsVersionToInstall: '${{ parameters.buildToolsVersionToInstall }}'
+                      tagName: '${{ parameters.tagName }}'
+                      interdependencyRange: '${{ parameters.interdependencyRange }}'
+                      packageTypesOverride: '${{ parameters.packageTypesOverride }}'
+              - template: /tools/pipelines/templates/include-build-lint.yml@self
+                parameters:
+                  taskBuild: '${{ parameters.taskBuild }}'
+                  taskLint: '${{ parameters.taskLint }}'
+                  taskLintName: '${{ parameters.taskLintName }}'
+                  buildDirectory: '${{ parameters.buildDirectory }}'
+              - '${{ if ne(convertToJson(parameters.taskTest), ''[]'') }}':
+                  - script: |
+                      echo "##vso[task.setvariable variable=startTest]true"
+                    displayName: Start Test
+                  - '${{ each taskTestStep in parameters.taskTest }}':
+                      - template: /tools/pipelines/templates/include-test-task.yml@self
+                        parameters:
+                          taskTestStep: '${{ taskTestStep }}'
+                          buildDirectory: '${{ parameters.buildDirectory }}'
+                  - '${{ if eq(variables[''testCoverage''], true) }}':
+                      - task: Bash@3
+                        displayName: Check for nyc/report directory
+                        inputs:
+                          targetType: inline
+                          workingDirectory: '${{ parameters.buildDirectory }}'
+                          script: >
+                            test -d nyc/report && echo '##vso[task.setvariable
+                            variable=ReportDirExists;]true' || echo 'No
+                            nyc/report directory'
+                        condition: >-
+                          and(succeededOrFailed(), eq(variables['startTest'],
+                          'true'))
```

Comment:
I get the tool tried to be smart about long lines but this is awful to read 😄 . Let's go back to single lines here, and probably in general where we see it changed to using `>` for concatting long strings.